ucm2/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -4,3 +4,26 @@ SectionUseCase."HiFi" {
 	File "HiFi.conf"
 	Comment "Play HiFi quality Music"
 }
+
+# the kcontrols initial values, which will be set by `alsactl init`
+
+BootSequence [
+	cset "name='Master Playback Switch' on"
+	cset "name='Master Playback Volume' 80"
+	cset "name='Capture Switch' on"
```

Comment:
@libinyang do you really want the switches on by default, or should they be set when an use cases needs capture or playback?